src/mem/AddrMapper.py

```diff
@@ -37,19 +37,19 @@
 from m5.SimObject import SimObject
 
 # An address mapper changes the packet addresses in going from the
-# slave port side of the mapper to the master port side. When the
-# slave port is queried for the address ranges, it also performs the
+# responder port side of the mapper to the requestor port side. When the
+# responder port is queried for the address ranges, it also performs the
 # necessary range updates. Note that snoop requests that travel from
-# the master port (i.e. the memory side) to the slave port are
+# the requestor port (i.e. the memory side) to the responder port are
 # currently not modified.
 class AddrMapper(SimObject):
     type = 'AddrMapper'
     cxx_header = 'mem/addr_mapper.hh'
     abstract = True
 
     # one port in each direction
-    master = MasterPort("Master port")
-    slave = SlavePort("Slave port")
+    requestor = RequestPort("Request port")
+    responder = ResponsePort("Response port")
```

Comment:
Maybe we should call this receiver? 

```suggestion
    receiver = ResponsePort("Port that receives requests and translates them before forwarding out of the requestor port.")
```